doc/030_preparing_a_new_repo.rst

```diff
@@ -474,6 +474,10 @@ The number of concurrent connections to the GCS service can be set with the
 ``-o gs.connections=10`` switch. By default, at most five parallel connections are
 established.
 
+The base URL to the GCS service can be set with the ``-o gs.baseurl=http://myhost/my/path/``
+switch. By default, the value is https://storage.googleapis.com/storage/v1/. This should not
+be confused with a full proxy like the ``HTTPS_PROXY`` environment variable.
```

Comment:
I've no clue how the base URL option relates to HTTPS_PROXY. Please remove that sentence.

Reply:
It was due to me initially using the HTTP proxy to proxy GCS and realizing I couldn't, but yes, can remove.